chapters/en/slides/module5/module5_04.Rmd

```diff
@@ -0,0 +1,348 @@
+
+Saturation is often used more as a stylistic choice
+to decide whether the colors we use should be muted/desaturated or vivid/saturated.
+
+---
+
+
+## Hues are used to distinguish categorical values
+
+```{python}
+# This code cell should be hidden but echo=False does not work with the knitr output hook
+import altair as alt
+from vega_datasets import data
+
+
+cars = data.cars()
+chart = alt.Chart(cars).mark_point(size=50, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None))
+
+(chart | chart.encode(alt.Color('Origin', scale=alt.Scale(scheme='blues'), title=None))).resolve_scale(color='independent')
+```
+
+Notes:
+Hue is useful to distinguish between categories,
+because it is often relatively easy for us to to say that different hues are distinct.
+We can see that blue, orange, red, etc are different and don’t easily mix them up.
+
+The leftmost plot uses different hues
+to separate the points belonging to different categories.
+We can quickly identify that there is a tota of three distinct colors being used here
+even without looking at the legend.
+
+In the plot to the right
+we used lightness within a single hue (blue) to label the different categories.
+Here it is much harder to say how many different categories there are,
+and even when we have the legend,
+it is difficult to tell which points are which shade of blue.
+
+---
+
+## It is often better to use established color schemes instead of making your own
+
+<center><img src=/module5/color-schemes.png></img></center>
+
+Notes:
+There are many different hues we could use to represent our colors
+and some might not go as well together as others,
+so how do we decide which ones to use?
+
+Fortunately,
+we don’t have to design our own combination of colors to use,
+but can pick from combinations designed by experts to be easy to tell apart
+and in most cases also suitable for people with color vision deficiencies.
+
+These color combinations are referred to as color schemes in Altair,
+but you might also hear them being called colormaps or color palettes.
+
+In this slide, you can see some of the color schemes for categorical values
+that are built into Altair.
+
+The default one is “tableau10”
+and it was named this way because it was originally designed by the company Tableau,
+but is now one of the most common categorical color schemes used in data visualization.
+[All Altair color schemes can be viewed here](https://vega.github.io/vega/docs/schemes/).
+
+---
+
+## Specifying color schemes in Altair
+
+```{python}
+alt.Chart(cars).mark_point(size=70, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None, scale=alt.Scale(scheme='set1')))
+```
+
+Notes:
+You can change the color scheme of a plot by specifying its name as a string
+to the `scheme` parameter inside `alt.Scale`.
+
+To easier see the changes we are making to the colors,
```

Comment:
```suggestion
To more easiely see the changes we are making to the colours,
```

Reply:
This is the last one I am going to fix, I am 99% sure we discussed that we were going to use Canadian spelling, so if I am correct, please continue to change this going forward. You might have to change it retroactively in this module (and others) as I was not paying attention to this earlier.

Reply:
yup, I will do find an replace on all afterwards.